roles/download/defaults/main.yml

```diff
@@ -142,8 +142,7 @@ kubectl_download_url: "https://storage.googleapis.com/kubernetes-release/release
 kubeadm_download_url: "https://storage.googleapis.com/kubernetes-release/release/{{ kubeadm_version }}/bin/linux/{{ image_arch }}/kubeadm"
 etcd_download_url: "https://github.com/etcd-io/etcd/releases/download/{{ etcd_version }}/etcd-{{ etcd_version }}-linux-{{ image_arch }}.tar.gz"
 cni_download_url: "https://github.com/containernetworking/plugins/releases/download/{{ cni_version }}/cni-plugins-linux-{{ image_arch }}-{{ cni_version }}.tgz"
-calicoctl_download_url: "https://github.com/projectcalico/calicoctl/releases/download/{{ calico_ctl_version }}/calicoctl-linux-{{ image_arch }}"
```

Comment:
Deleting this breaks older versions of calico because project calico did this in the middle of a release stream and have not provided redirects from the old urls or any other compatibility mechanism.

We can swap the urls between the old and the new ones but until we remove all older versions that kubespray supports we cannot do this.

Reply:
I swapped the urls.